certora/spec/timelockAuthorizerAlex.spec

```diff
@@ -179,12 +180,12 @@ rule onlyPendingRootCanBecomeNewRoot(method f, env e){
 
     address root_ = _root();
     
-    assert root_ == _root,
+    assert root_ == _root,    // this is verified ?  can't root change? 
     // assert root_ == _root || root_ == _pendingRoot,
         "root can either remain unchanged or change to the pendingRoot";
 }
 
-
+// sanity rules from here, right? you can remove or move th the main spec and document that it is sanity 
```

Comment:
it's not in the main spec. I keep our specs as they were in case we continue with this contract later